404.html

```diff
@@ -4,17 +4,20 @@
 <!doctype html>
 <html>
 	<head>
-		<link rel="stylesheet" href="stylesheet.css"/>
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
 		<link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"/>
 	</head>
-	<body>
-    <img src="logo.png">
-
-    <div class="boxes">
-	        <h1>404</h1>
-		<p>Nothing's better than a nice MLT - mutton, lettuce, and tomato sandwich when the mutton's nice and leeeean, and the tomatoes are ripe - mmh!</p>
-		<p>Seems like you've gotten another 404. Ah well, that page doesn't exist. <a href="/">Here</a>'s a page that does exist.</p>
-    </div>
-		
+<body>
+		<center>
+			<br/>
+			<div class="jumbotron">
+				<h2> 404 </h2>
+				<h3> You seem a bit lost, there doesn't seem to be anything here... </h3>
```

Comment:
Oh, and could you change this string back? I liked the MLT quote.

Reply:
Okay 👍